client/webpack.config.js

```diff
@@ -376,13 +376,14 @@ const webpackConfig = {
 			: [] ),
 
 		/*
-		 * When not available, replace languages-meta.json with fallback-languages-meta.json.
+		 * When available, replace fallback-languages-meta.json with languages-meta.json
 		 */
 		hasLanguagesMeta &&
 			new webpack.NormalModuleReplacementPlugin(
-				/^languages[/\\]fallback-languages-meta.json$/,
-				'languages/languages-meta.json'
+				new RegExp( path.join( __dirname, 'languages/fallback-languages-meta.json' ) ),
+				'./languages-meta.json'
```

Comment:
Hmm, interesting. This will only work if the importing file lives in the same directory as `languages-meta.json`, whereas the previous approach will only work if the language files live in `languages/`. I'm not sure which approach is safer.

Reply:
Since we're harcoding `languages/` in the previous line, perhaps hardcoding it here makes more sense too?